AI-and-Analytics/End-to-end-Workloads/Census/sample.json

```diff
@@ -11,7 +11,7 @@
   "ciTests": {
   	"linux": [
     {
-  		"env": ["source /opt/intel/oneapi/setvars.sh --force", "source activate base", "conda create --name intel-aikit-modin -c intel/label/oneapibeta -c intel -c conda-forge runipy intel-aikit-modin=2021.1b10", "conda activate intel-aikit-modin"],
+  		"env": ["source /opt/intel/oneapi/setvars.sh --force", "conda create -n aikit-modin-test -c intel/label/oneapibeta -c intel -c conda-forge runipy intel-aikit-modin", "conda activate aikit-modin-test"],
```

Comment:
conda is waiting for the response from user. We could avoid such behavior by adding ‘-y’ option, which will automatically send yes.

Reply:
Yea just made that change